src/test/java/teammates/test/cases/ui/browsertests/InstructorCommentsPageUiTest.java

```diff
@@ -102,11 +102,13 @@ private void testScripts() throws Exception {
         commentsPage.verifyContains("comments.idOfArchivedCourse");
         
         commentsPage.clickNextCourseLink();
-        assertTrue("URL: " + browser.driver.getCurrentUrl(), browser.driver.getCurrentUrl().contains(Const.ActionURIs.INSTRUCTOR_COMMENTS_PAGE
+        assertTrue("URL: " + browser.driver.getCurrentUrl(),
+                   browser.driver.getCurrentUrl().contains(Const.ActionURIs.INSTRUCTOR_COMMENTS_PAGE
                    + "?user=comments.idOfInstructor1OfCourse1&courseid=comments.idOfArchivedCourse"));
         
         commentsPage.clickPreviousCourseLink();
-        assertTrue("URL: " + browser.driver.getCurrentUrl(), browser.driver.getCurrentUrl().contains(Const.ActionURIs.INSTRUCTOR_COMMENTS_PAGE
+        assertTrue("URL: " + browser.driver.getCurrentUrl(),
+                   browser.driver.getCurrentUrl().contains(Const.ActionURIs.INSTRUCTOR_COMMENTS_PAGE
                    + "?user=comments.idOfInstructor1OfCourse1&courseid=comments.idOfTypicalCourse1"));
```

Comment:
Same here, indent 4 spaces to the right due to it only being part of a parameter.